distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Of_Day.enso

```diff
@@ -257,37 +257,57 @@ type Time_Of_Day
        Add the specified amount of time to this instant to get a new instant.
 
        Arguments:
-       - amount: The amount of time to add to this instant. Can be only
-         `Duration`.
+       - amount: The amount of time to add to this instant. Can be `Duration` or
```

Comment:
```suggestion
       - amount: The amount of time to add to this instant. It can be `Duration` or
```

I know this was already there, but I think this should be a full sentence, so if you could please amend this 🙂

Reply:
Done.